modules/apps/calendar/calendar-test/src/testIntegration/java/com/liferay/calendar/upgrade/v4_2_1/test/UpgradeCalendarBookingTest.java

```diff
@@ -39,7 +47,14 @@
  * @author István András Dézsi
  */
 @RunWith(Arquillian.class)
-public class UpgradeCalendarBookingTest {
+public class UpgradeCalendarBookingTest extends BaseCTUpgradeProcessTestCase {
```

Comment:
@rijema since we are going to rename it to something like `CalendarBookingCTUpgradeProcessTest`, can you create a different class instead of reusing this existing one? 

You can move the common code to a Util class `CalendarBookingUpgradeProcessTestUtil`